src/System.Diagnostics.DiagnosticSource/tests/DiagnosticSourceEventSourceBridgeTests.cs

```diff
@@ -15,7 +15,7 @@
 namespace System.Diagnostics.Tests
 {
     //Complex types are not supported on EventSource for .NET 4.5
-    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework45)]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Net45 | TargetFrameworkMonikers.Net451 | TargetFrameworkMonikers.Net452)]
```

Comment:
I wanted to leave the option for having version markers but for corefx I don't think we need to maintain these so I would just remove the test completely to try and simplify our matrix.